pages/index.js

```diff
@@ -2,79 +2,180 @@ import styled from "styled-components";
 import Form from "@/components/Form/Form.js";
 import ShoppingItem from "@/components/ShoppingItem/ShoppingItem.js";
 import { useState } from "react";
-import { categories } from "@/lib/categoriesData";
+// import { categories } from "@/lib/categoriesData";
+import Image from "next/image";
+
 export default function ShoppingItemsList({
   shoppingItemsWithCategoryColor,
+  // backgroundColor,
+  // shoppingItems,
   onAddItem,
   onEditItem,
   onDeleteItem,
+  onToggleIsPurchased,
+  purchasedItems,
+  unpurchasedItems,
 }) {
-  const [mode, setMode] = useState("");
+  const [mode, setMode] = useState("default");
 
   const [editedItem, setEditedItem] = useState({});
 
+  // const [isPurchased, setIsPurchased] = useState("");
+
   function handleChangeMode(mode) {
     setMode(mode);
   }
 
   return (
     <>
       <StyledHeader>
-        <h1>Shopping Car-d</h1>
-      </StyledHeader>
+        <h1>
+          Tot
+          <span style={{ fontSize: 30, color: "green", letterSpacing: "-8px" }}>
+            e
+          </span>
+          &#39;
+          <span style={{ fontSize: 42 }}>A</span>lly
+          <br></br>
+          <span
+            style={{
+              fontSize: 20,
+              color: "green",
+            }}
+          >
+            ...baggin&#39; it.
+          </span>
+        </h1>
 
+        <Image
+          alt="drawing of a smiling tote bag"
+          // replace src:
+          src={"/images/tote-bag_02.png"}
+          // src={"/images/tote-bag_01.png"}
+          style={{ objectFit: "contain" }}
+          width={80}
+          height={80}
+        />
+      </StyledHeader>
       <main>
-        {shoppingItemsWithCategoryColor.length === 0 && (
-          <StyledNoItemsMessage>
-            There are no items on your shopping list. Add items using the form
-            below.
-          </StyledNoItemsMessage>
+        {/* >>> try to use this: */}
+        {/* {mode === "default" && ( */}
+        {mode !== "edit" && mode !== "add" && (
```

Comment:
You only put my suggestion as a comment but did not implement it in the code:

```suggestion
        {mode === "default" && (
```

Reply:
Done earlier